app/src/main/java/com/deuvox/deuvoxapp/feature/main/splashscreen/SplashScreenViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package com.deuvox.deuvoxapp.feature.main.splashscreen
+
+import android.util.Log
+import androidx.lifecycle.viewModelScope
+import com.deuvox.deuvoxapp.base.viewmodel.BaseViewModel
+import com.deuvox.domain.GetDataUseCase
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+class SplashScreenViewModel @Inject constructor(
+    private val getDataUseCase: GetDataUseCase
```

Comment:
override onDispose method, and dispose the usecase

Reply:
how to dispose the usecase

Reply:
or i shouldn't create the viewmodel ?

Reply:
ohh it's okay, implement just implement in `DRP-6`